superset-frontend/src/dashboard/actions/nativeFilters.ts

```diff
@@ -103,6 +103,20 @@ export interface SetExtraFormData {
   currentState: CurrentFilterState;
 }
 
+export const SAVE_FILTERS_SET = 'SAVE_FILTERS_SET';
```

Comment:
I believe it should be "filter set"/"filter sets", not "filters set"/"filters sets" (This comment applies to all occurrences of "filters set")

Reply:
I also thought about naming, but this action save 1 set of filters, so I did it as `FILTERS_SET`

Reply:
It's not super critical, but for this feature the pluralization inconsistency is starting to show